packages/composite-checkout/src/components/checkout-payment-methods.js

```diff
@@ -65,14 +70,22 @@ CheckoutPaymentMethods.propTypes = {
 	onChange: PropTypes.func.isRequired,
 };
 
-function PaymentMethod( { id, LabelComponent, PaymentMethodComponent, checked, onClick } ) {
+function PaymentMethod( {
+	id,
+	LabelComponent,
+	PaymentMethodComponent,
+	checked,
+	onClick,
+	ariaLabel,
+} ) {
 	return (
 		<RadioButton
 			name="paymentMethod"
 			value={ id }
 			id={ id }
 			checked={ checked }
 			onChange={ onClick ? () => onClick( id ) : null }
+            ariaLabel={ ariaLabel }
```

Comment:
Something seems weird about the indentation here. I think you used spaces instead of tabs. I'm surprised Prettier didn't fix this during the commit. 🤔

Reply:
Weird! I think there was a sort of race condition involving prettier, my IDE's smart indentation settings, and interactive rebase.